src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -62,12 +64,24 @@ public void execute_commandExecutionError_throwsCommandException() {
         assertCommandException(deleteCommand, MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
     }
 
+    @Test
+    public void execute_commandExecutionError2_throwsCommandException() {
```

Comment:
Perhaps a more meaningful name here? Best to avoid using numbers as it's a little ambiguous. Similar for the other method in this class.
Examples can be, execute_listEventCommandExecutionError_throwsCommandException()

Reply:
Edited, thanks!